pyslackersweb/website/tasks.py

```diff
@@ -31,111 +41,98 @@ class Channel:
     members: int
 
 
-def sync_github_repositories(app: web.Application) -> Callable[[], Awaitable[None]]:
-    session = app["client_session"]
-
-    async def _sync_github() -> None:
-        logger.debug("Refreshing GitHub cache")
-        try:
-            async with session.get(
-                "https://api.github.com/orgs/pyslackers/repos",
-                headers={"Accept": "application/vnd.github.mercy-preview+json"},
-            ) as r:
-                repos = await r.json()
-
-            repositories = []
-            for repo in repos:
-                if repo["archived"]:
-                    continue
-
-                repositories.append(
-                    Repository(
-                        repo["name"],
-                        repo["description"],
-                        repo["html_url"],
-                        repo["stargazers_count"],
-                        repo["topics"],
-                    )
+async def sync_github_repositories(
+    session: ClientSession, redis: AbcConnection, *, cache_key: str = GITHUB_REPO_CACHE_KEY
```

Comment:
I don't like the `AbcConnection` typing. It's not clear where it comes from, but `aioredis.abc.AbcConnection` is a bit long

Reply:
I'll alias it